### PR TITLE
RHEL-8.7+/9.1+: replace RHSM config on EC2 RHUI images with `redhat-cloud-client-configuration` package

### PR DESCRIPTION
**Pull request by thozza**
- [tools: update repos used to generate 8.7 and 9.1 image manifests](https://github.com/osbuild/osbuild-composer/commit/826a8b32cfefa4f43fab135f6c55a07155a24982)
- [distro/rhel8: adjust RHSM-related config on EC2 8.7+ images](https://github.com/osbuild/osbuild-composer/commit/77befd04d9cccd8760433b3758a327dbea89da5b)
- [distro/rhel9: adjust RHSM-related config on EC2 9.1+ images](https://github.com/osbuild/osbuild-composer/commit/d68d7f40bc3c003aead0136c3c58cf37343ad7e5)

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Comment by thozza** [comment]
Note that these changes have **Critical** priority.

File: internal/distro/rhel9/package_sets.go [hunk 1]
```diff
@@ -585,9 +587,19 @@ func ec2CommonPackageSet(t *imageType) rpmmd.PackageSet {
 	}.Append(bootPackageSet(t)).Append(coreOsCommonPackageSet(t)).Append(distroSpecificPackageSet(t))
 }
 
+// common rhel ec2 RHUI image package set
+func rhelEc2CommonPackageSet(t *imageType) rpmmd.PackageSet {
+	ps := ec2CommonPackageSet(t)
+	// Include "redhat-cloud-client-configuration" on 9.1+ (COMPOSER-1805)
+	if !common.VersionLessThan(t.arch.distro.osVersion, "8.7") {
```
Comment (achilleas-k) [inline internal/distro/rhel9/package_sets.go:594]: `s/8.7/9.1/`?

**Review by achilleas-k — DISMISSED** [review]

File: internal/distro/rhel9/package_sets.go (hunk 1 above)
Comment (thozza) [reply internal/distro/rhel9/package_sets.go:594]: 😱  thanks!

**Comment by thozza** [comment]
Fixed the review issue and rebased on `main`.

**Review by ondrejbudai — APPROVED** [review]
Lovely, thanks for such a quick implementation!

**Pull request merged**